packages/wallet/src/services/DelegationTracker/RewardAccounts.ts

```diff
@@ -22,13 +22,15 @@ export const createQueryStakePoolsProvider =
       store.getValues(poolIds),
       coldObservableProvider(
         () =>
-          stakePoolSearchProvider.queryStakePools({
-            filters: { identifier: { values: poolIds.map((poolId) => ({ id: poolId })) } }
-          }),
+          stakePoolSearchProvider
+            .queryStakePools({
+              filters: { identifier: { values: poolIds.map((poolId) => ({ id: poolId })) } }
+            })
+            .then(({ pageResults }) => pageResults),
```

Comment:
We need to handle the `totalCount` too, which is needed for pagination. Please review the application implementation to guide us 👍

Reply:
Maybe we can return an object with this structure:
```
{
   totalCount: number,
   stakePools: KeyValueStore<Cardano.PoolId, Cardano.StakePool>
}
```

Reply:
Actually, ignore this feedback :smile:. `totalCount` is not required in this context, as it's here to simply cache the stake pool the wallet is delegated to. The app interfaces directly with the provider for search results, so can handle the new return type as it sees fit.